USER_MANUAL.md

```diff
@@ -1549,58 +1740,983 @@ The following figure offers a visual guideline for the various parameters that c
+- [emissions_rate_source](#emissions_rate_source-source-file)
+- [duration](#duration-source-file)
+- [multiple_emissions_per_source](#multiple_emissions_per_source-source-file)
+
+Method-specific columns are values that can be defined for a particular method. To use them, replace the `{method}` placeholder with the relevant [method_name](#method_name) that the user is parameterizing. The following method-specific columns are available for the Sites file:
+
+- {method}_[spatial](#method_spatial-source-file)
+  
+As mentioned above, if the optional columns are not provided, they will be populated with any existing higher-level values. For example, if the duration column is missing but is provided in the virtual world parameter file, LDAR-Sim will populate the durations of the specified component-source with the relevant durations from the virtual world parameter file.
+
+**Note:** As with most name-related parameters in LDAR-Sim, the values in the `component` and `emissions_rate_source` columns are case-sensitive and must be consistent across all related files. Any method specific parameters also require the same care with the case-sensitivity and consistency in all relevant files with the method name.
+
+See [source file](#source-file) for details on how to set the parameter for simulation.
+
+--------------------------------------------------------------------------------
+
+Below is an example of a _source file_ that defines two different components and their relevant sources.
+
+|component|source|emissions_rate_source|repairable|
+|----------|----------|----------|----------|
+|tank|fugitive|fugitive_rates|TRUE|
+|tank|non-repairable|operational|FALSE|
+|well|fugitive|fugitive_rates|TRUE|
+
+--------------------------------------------------------------------------------
+
+#### component (Source File)
+
+**Description:** A user-defined component represents the smallest item that the highest resolution technology would be able to decipher and detect emissions from in real life.
+
+**Note:** A `component`can have multiple emission sources, each potentially producing emissions at the same time. A _component level_ [measurement strategy](#measurement_scale) measures the total sum of all emissions from these sources at any given moment.
+
+**Note of caution:** The `component` and `source` columns together act as a compound key. In simple terms, each component/source pair must be unique and exist only once.
+
+This column is case-sensitive and must match the corresponding columns provided in other infrastructure files.
+
+#### source (Source File)
+
+**Description:** A user-defined source represents a potential emission source for a given component. It is the smallest granular level considered for emission production.
+
+**Note:** The `source` column is often identical to the `emissions_rate_source` column. However it is in there incase users would like to use the same `emission_rate_sources`, for different `sources`.
```

Comment:
Can we make it clear that the values in the source column are distinct from the emissions rate source column and you do need to populate both to use an emissions rate source